blt/urls.py

```diff
@@ -506,6 +507,7 @@
     path("change_bid_status/", change_bid_status, name="change_bid_status"),
     path("fetch-current-bid/", fetch_current_bid, name="fetch_current_bid"),
     path("Submitpr/", submit_pr, name="submit_pr"),
+    path("AutoLabel/", AutoLabel, name="AutoLabel"),
```

Comment:
Could you use something like
```suggestion
    path("issue-auto-label/", AutoLabel, name="AutoLabel"),
```